Doc/library/sqlite3.rst

```diff
@@ -1365,41 +1398,83 @@ timestamp converter.
 
 .. _sqlite3-controlling-transactions:
 
-Controlling Transactions
-------------------------
+Controlling Transactions Using the Autocommit Property
```

Comment:
Property or Attribute?

Reply:
I guess attribute is more correct. I'll make sure that term is used consistently.